pages/cadastro/index.public.js

```diff
@@ -1,7 +1,7 @@
 import { useState, useRef } from 'react';
 import { useRouter } from 'next/router';
 import { DefaultLayout, PasswordInput } from 'pages/interface/index.js';
```

Comment:
```suggestion
import { DefaultLayout, Link, PasswordInput } from 'pages/interface/index.js';
```

Use o `Link` customizado